cartridge/graphql/execute.lua

```diff
@@ -194,12 +194,12 @@ local function completeValue(fieldType, result, subSelections, context, opts)
   end
 
   if fieldTypeName == 'List' then
-    local innerType = fieldType.ofType
-
-    if type(result) ~= 'table' then
-      error('Expected a table for ' .. innerType.name .. ' list')
+    if type(result) ~= 'table' or not util.is_array(result) then
```

Comment:
maybe simplify to `if not util.is_array(result) then`?

Reply:
It makes us to patch is_array function a bit. But it seems to be ok.